NEWS.md

```diff
@@ -11,7 +11,6 @@
 
 - Internal changes only.
 
-
```

Comment:
I want an empty line here, contrary to Markdown style guides.

Reply:
Added :heavy_check_mark: